.travis.yml

```diff
@@ -5,4 +5,12 @@ install:
   - npm install bower gulp -g
   - npm install && bower install
 script:
-  - gulp test
+  - gulp test --psc-rts-opts -N4
+after_success:
+- >-
+  test $TRAVIS_TAG &&
+  ./node_modules/.bin/psc-publish > .pursuit.json &&
+  curl -X POST http://pursuit.purescript.org/packages \
+    -d @.pursuit.json \
+    -H 'Accept: application/json' \
+    -H "Authorization: token ${GITHUB_TOKEN}"
```

Comment:
When we add this we also need to add the `GITHUB_TOKEN` in the build settings for the repo in travis. There's [some info here](http://pursuit.purescript.org/help) about generating the token under "Submitting packages from a script" - no need to do it for this repo though, I've set it up already.